package.json

```diff
@@ -1,13 +1,19 @@
 {
+  "scripts": {
+    "test": "glob -c \"node --loader tsx --no-warnings --test\" \"./test/**/*_test.ts\""
```

Comment:
This approach will resolve https://github.com/kachick/renovate-config-asdf/pull/1027#discussion_r1271377029